File: algorithms/650. 2 Keys Keyboard/solution.py
class Solution(object):
    def minSteps(self, n):
        """
        :type n: int
        :rtype: int
        """
        s, f = 0, 2
        while f <= n:
            while n % f == 0:
                s += f
                n /= f
            f += 1
        return s

# 126 / 126 test cases passed.
# Status: Accepted
# Runtime: 42 ms
# beats 66.22 %

File: algorithms/650. 2 Keys Keyboard/solution.cc
class Solution {
public:
    int minSteps(int n) {
        int s = 0;
        for ( int i = 2; i <= n; ++i )
            while ( n % i == 0 ) {
                s += i;
                n /= i;
            }
        return s;
    }
};

// 126 / 126 test cases passed.
// Status: Accepted
// Runtime: 3 ms
// beats 62.38 %
